kiwi/volume_manager/btrfs.py

```diff
@@ -358,18 +358,22 @@ def _xml_pretty(self, toplevel_element):
         xml_data_domtree = minidom.parseString(xml_data_unformatted)
         return xml_data_domtree.toprettyxml(indent="    ")
 
-    def _create_quota_group_info(self):
+    def _create_snapper_configuration(self):
```

Comment:
Misleading function name?

Reply:
what about `_create_snapper_quota_configuration`? I want to add snapper into the name because all the changes done here are snapper specific.